### ISSUE TITLE
gettext does not extract definition terms

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#975]
When creating a definition list:

```
Some term
      The corresponding definition

Some other term
      The corresponding definition #2
```

The definition terms ("Some term" and "Some other term") are not extracted and thus cannot be translated.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/975
- Originally reported by: Xavier Perseguers
- Originally created at: 2012-07-25T09:15:07.716

**Comment by shimizukawa** [comment]
_From Nozomu Kaneko on 2012-12-31 13:11:40+00:00_

This issue is already fixed, right?

ecb7ed83c13e240ea79d879192182b97774fdbcd

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2012-12-31 17:09:36+00:00_

It was proposed by pull request #72 and was merged. I'm going to add test for this.

**Comment by shimizukawa** [comment]
_From Takayuki Shimizukawa on 2013-01-01 06:33:43+00:00_

Add regression test for gettext does not extract definition terms. Closes #975

→ <<cset 7022b6f2519030d0928690b5fd61fd78c0974f12>>
